ci/format_pre.sh

```diff
@@ -58,6 +58,9 @@ bazel run "${BAZEL_BUILD_OPTIONS[@]}" //tools/code_format:python_check -- --diff
 CURRENT=extensions
 bazel run "${BAZEL_BUILD_OPTIONS[@]}" //tools/extensions:extensions_check
 
+CURRENT=spelling
+"${ENVOY_SRCDIR}"/tools/spelling/check_spelling_pedantic.py --mark check
```

Comment:
why delete the other spell check?  I thought they caught different things?

Reply:
afaict one is not catching anything

when this was discussed in the maintainer channel it was ~agreed to remove

Reply:
ah I must have been out